### PR TITLE
feat: update config file and add config generator

### PR DESCRIPTION
**Pull request by isaackps**
# In this PR:
- Updated config file docs
- added a section for config generation with oa-cli

https://www.pivotaltracker.com/n/projects/2424361

### CODE REVIEW

File: docs/document-creator/config-file/config-generator.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Alternatively, we have a tool to help generate the config file.
+
+Before we proceed, you will need to have Open Attestation CLI v1.39.3 or higher installed in your computer.
+
+## Step 1: Acquiring a wallet.json file
+
+Firstly, we will need to acquire a wallet.json file, please refer [here](https://www.openattestation.com/docs/verifiable-document/wallet) on how to create the wallet.json file.
+
+If you have already have a wallet.json file, you can skip this step and proceed to [step 2](#step-2).
```
Comment (ryanseee) [inline docs/document-creator/config-file/config-generator.md:17]:
```suggestion
If you already have a wallet.json file, you can skip this step and proceed to [step 2](#step-2).
```

File: docs/document-creator/config-file/config-generator.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Run this command to generate a config file that includes some generic template forms.
+
+```
+open-attestation config create --output-dir <./examples/config> --encrypted-wallet-path </path/to/wallet.json> --config-type tradetrust
+
+```
+
+During the creation, you will be prompted for a password. Make sure to remember it for the following steps.
+
+The option `--output-dir` specify which directory the config file will be created in.
```
Comment (ryanseee) [inline docs/document-creator/config-file/config-generator.md:30]:
```suggestion
The option `--output-dir` specifies which directory the config file will be created in.
```

File: docs/document-creator/config-file/config-generator.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+The option `--encrypted-wallet-path` indicates a path to the encrypted wallet that you acquire in step 1.
```
Comment (ryanseee) [inline docs/document-creator/config-file/config-generator.md:32]:
```suggestion
The option `--encrypted-wallet-path` indicates a path to the encrypted wallet that you acquired in step 1.
```

File: docs/document-creator/config-file/config-generator.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+The option `--config-type` specify which default template to use to create the config file. For the case above, we specify to use tradetrust default template to create the config file.
```
Comment (ryanseee) [inline docs/document-creator/config-file/config-generator.md:34]:
```suggestion
The option `--config-type` specifies which default template to use to create the config file. For the case above, we specify to use tradetrust default template to create the config file.
```

File: docs/document-creator/config-file/config-generator.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+Once the config file is generated, you can test it out on our [document creator website](https://creator.tradetrust.io/).
+
+To take it a step further, you can make it your own config file by updating the forms in the forms section with your forms, so you can create and issue your documents.
```
Comment (ryanseee) [inline docs/document-creator/config-file/config-generator.md:38]:
```suggestion
To take it a step further, you can customise your config file by **replacing the forms in the forms section with your own customised forms**, so you can create and issue your own documents.
```
would the words in bold be better? so the word 'forms' wont seem like its repeated so close to each other haha.

File: docs/document-creator/config-file/config-generator.md (hunk 1 above)
Comment (isaackps) [reply docs/document-creator/config-file/config-generator.md:17]: ok

File: docs/document-creator/config-file/config-generator.md (hunk 2 above)
Comment (isaackps) [reply docs/document-creator/config-file/config-generator.md:30]: ok

File: docs/document-creator/config-file/config-generator.md (hunk 3 above)
Comment (isaackps) [reply docs/document-creator/config-file/config-generator.md:32]: ok

File: docs/document-creator/config-file/config-generator.md (hunk 5 above)
Comment (isaackps) [reply docs/document-creator/config-file/config-generator.md:38]: haha ok

**Review by ryanseee — APPROVED** [review]
lgtm

**Review by ryanseee — APPROVED** [review]
lgtm

**Pull request merged**